### ISSUE TITLE
This domain name conflicts with an existing zone, or is disallowed by policy.

### ISSUE THREAD

**Issue by pjhfggij** [desec-io/desec-stack#905]
can't create new domains anymore

random domain like: 
aieurbgueibg.denyn.io
suirbge.desec.org
eirbgukeisbgkiesrg.desec.io

will yield this error:
This domain name conflicts with an existing zone, or is disallowed by policy.

**Comment by peterthomassen** [comment]
Right. It's currently disallowed by policy, as the message says. For context, please see https://talk.desec.io/t/dyndns-service-down/823/3.

**Comment by pjhfggij** [comment]
thanks @peterthomassen for explaining, that makes sense. I'm looking forward then when it will be enabled again. It's such a shame that people take advantage of your service.

I was planning to redeploy a matrix server from scratch, while solving a non-standard UEFI boot issues when this hit me... in other words it wasn't my day :)
